runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -1730,8 +1730,8 @@ class INTERPRETER_CLASS
 						_currentThread,
 						J9OBJECT_CLAZZ(_currentThread, *(j9object_t*)_arg0EA),
 						&romMethod->nameAndSignature,
-						NULL,
-						J9_LOOK_VIRTUAL | J9_LOOK_NO_THROW);
+						methodClass,
```

Comment:
This isn't used by your changes anymore. Removing it will require you to move the check in processMethod above line 160.